src/lib/decodeBySigHash.ts

```diff
@@ -44,16 +44,49 @@ export type FetchResult = {
   hex_signature: string;
 };
 
+// @internal
+// there are more types, but we don't need them for now
+export enum HexSigType {
```

Comment:
We've decided that we would rather try to avoid enums in favor of string literal types, @hasparus wdyt?

Reply:
+1